src/Gnocco/GncPCGRandomGenerator.class.st

```diff
@@ -0,0 +1,163 @@
+Class {
+	#name : #GncPCGRandomGenerator,
+	#superclass : #Object,
+	#instVars : [
+		'state',
+		'seed'
+	],
+	#category : #'Gnocco-Grammar'
+}
```

Comment:
I think this class is small enough, we can have a self-contained implementation.

Because we depend on the distribution of this random generator for the tests.
=> Using another random generator would break them

Thus, if the default one in Pharo changes it's behaviour (nothing really strange, it happened in the last year at least once :)) we are covered.

Also, this allows gnocco to not depend on polymath and run in different Pharo versions!